### PR TITLE
Lost heartbeat schedule notification + Fixes & improvements

### PR DESCRIPTION
**Pull request by Fralez**
- Schedule notifications done
- LocalNotificationIds system created
- Re-random daily biometric notification improved
- Removed mistakenly used doc comments

### CODE REVIEW

File: lib/services/LocalNotificationService.dart [hunk 1]
```diff
@@ -9,6 +9,18 @@ AndroidInitializationSettings initializationSettingsAndroid;
...
+/// Notification IDs stored in an enum - access the value using `.index` accessor within enum elements
+enum LocalNotificationIds {
+  morningDaily,
+  middayDaily,
+  afternoonDaily,
+  outOfZoneInstant,
+  lostHeartbeatScheduled,
+  lostConnectivityInstant,
+  lostGpsInstant,
+  testSchedule
```
Comment (Fralez) [inline lib/services/LocalNotificationService.dart:21]: Forgot to delete this, I will be updating ASAP

File: lib/components/pages/home_page.dart [hunk 2]
```diff
@@ -67,8 +67,10 @@ class _HomePageState extends State<HomePage> {
     initializeLocalNotifications(_onDidReceiveLocalNotification,
         onSelectNotification: (String payload) {
       if (payload.startsWith("doBiometrics")) {
-        locator<NavigationService>()
-            .navigateTo('alertpagebiometrics', payload.split("-")[1]);
+        locator<NavigationService>().navigateTo(
+            'alertpagebiometrics',
+            LocalNotificationIds.values
```
Comment (cherta) [inline lib/components/pages/home_page.dart:72]: Same way you have an enum I think you can provide a method that abstracts all this ugliness (?), something similar to `LocalNotificationIds.getId(payload)`

File: lib/services/BiometricAuthService.dart [hunk 3]
```diff
@@ -17,28 +20,54 @@ Future<bool> authenticateFingerprint(int notificationId) async {
...
-    resetDailyNotificationTime(
-        2, hours, minutes, 00, 'doBiometrics-$notificationId');
+/// Makes a re-random of sthe next daily notification using the following logic:
+///
+/// If [id] == `morningDaily` -> Re-random `LocalNotificationIds.middayDaily`
+///
+/// If [id] == `middayDaily` -> Re-random `LocalNotificationIds.afternoonDaily`
+///
+/// If [id] == `afternoonDaily` -> Re-random `LocalNotificationIds.morningDaily`
+void reRandomDailyNotifications(notificationId) {
```
Comment (cherta) [inline lib/services/BiometricAuthService.dart:34]: I'll change the method name to something less cryptic like `reOrderDailyNotifications`

**Review by cherta — APPROVED** [review]
The code is well done, I still haven't wrapped my head around the `generateScheduledNotification` in `lib/services/GeofencingService.dart`, maybe we can sync offline about those changes, I would like to understand the intention behind the code.

**Pull request merged**